src/actions/dashboardActions.ts

```diff
@@ -87,6 +89,21 @@ export function saveDashboard(id: number, name: String, contents: DashboardModel
   }
 }
 
+export function addWidgetAfter() {
+  return {
+    type: ADD_WIDGET,
```

Comment:
not sure if this is just a skeleton but I think this might require a param as well (this can come later)